articles/active-directory/devices/howto-vm-sign-in-azure-ad-windows.md

```diff
@@ -130,6 +130,8 @@ Finally, install the Azure AD login VM extension to enable Azure AD login for Wi
 > [!NOTE]
 > You can install AADLoginForWindows extension on an existing Windows Server 2019 or Windows 10 1809 and later VM to enable it for Azure AD authentication. An example of AZ CLI is shown below.
 
+### [Azure CLI](#tab/azure-cli)
```

Comment:
Should not be in link format. Just ### Azure CLI

Reply:
@MicrosoftGuyJFlo , The goal was to get the PowerShell code to appear in a tab side-by-side like in the link below so it doesn't unnecessarily add to the length of the document.

https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/programmatically-create-subscription?tabs=azure-powershell#find-accounts-you-have-access-to

Reply:
@MicrosoftGuyJFlo - Could you indicate whether your requested revision is changed by feedback from @Ayanmullick?